sdk/hdinsight/Azure.ResourceManager.HDInsight/tests/HDInsightManagementTestBase.cs

```diff
@@ -20,12 +20,11 @@ public class HDInsightManagementTestBase : ManagementRecordedTestBase<HDInsightM
     {
         protected ArmClient Client { get; private set; }
         protected const string DefaultResourceGroupPrefix = "HDInsightRG-";
-        protected AzureLocation DefaultLocation = AzureLocation.EastUS;
+        protected AzureLocation DefaultLocation = AzureLocation.EastAsia;
         protected const string Common_User = "sshuser5951";
         protected const string Common_Password = "Password!5951";
-        protected const string Common_VNet_Id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/yuchen-ps-test/providers/Microsoft.Network/virtualNetworks/hdi-vn-0";
-        protected const string Common_SubNet = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/yuchen-ps-test/providers/Microsoft.Network/virtualNetworks/hdi-vn-0/subnets/default";
-
+        protected const string Common_VNet_Id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/yukundemo1/providers/Microsoft.Network/virtualNetworks/yk01networkeastasia";
```

Comment:
let us use common name to exclude any credentials.
The resource group name is hdi-ps-test
the vnet name starts hditestvnet